Cargo.toml

```
@@ -410,109 +410,109 @@ zip = { workspace = true, optional = true }
 uuhelp_parser = { optional = true, version = ">=0.0.19", path = "src/uuhelp_parser" }
```

I guess here we can also use `0.2.0`.
```suggestion
uuhelp_parser = { optional = true, version = "0.2.0", path = "src/uuhelp_parser" }
```